.github/workflows/deploy-clusters.yaml

```diff
@@ -75,18 +75,19 @@ jobs:
         webhook: ${{ secrets.DISCORD_WEBHOOK }}
         title: "charon-gnosis-0"
 
-    - name: Delete charon-prater-0
-      run: ./delete-cluster.sh charon-prater-0
-      shell: bash
-    - name: Deploy charon-prater-0
-      run: ./deploy-cluster-with-charon-tag.sh charon-prater-0 ${{ steps.charon_image_tag.outputs.tag }}
-      shell: bash
+    # Disable charon-prater-0 automation until web3signer experiment is completed
```

Comment:
Are we removing prater 0?

Reply:
this is temp until we are good with the web3signer PoC results